.golangci.yml

```diff
@@ -82,3 +82,9 @@ linters-settings:
       - name: unhandled-error
         arguments:
           - "fmt.Printf"
+issues:
+  exclude-rules:
+    - path: tests\/.+\.go
```

Comment:
This will capture tests with `tests` somewhere in the path, but it won't capture the vast majority of  _test.go unit tests that we have. Is that intended?

Reply:
I want this to get applied to functional tests in `tests` dir only. But you are right, I will capture others too. I need to add prefix it with `^`.